control_toolbox/include/control_toolbox/pid.hpp

```diff
@@ -34,12 +34,84 @@
 #define CONTROL_TOOLBOX__PID_HPP_
 
 #include <chrono>
+#include <cstdint>
+#include <limits>
+#include <string>
 
 #include "rclcpp/duration.hpp"
 #include "realtime_tools/realtime_buffer.hpp"
 
 namespace control_toolbox
 {
+class AntiwindupStrategy
+{
+public:
+  enum Value : int8_t
+  {
+    NONE = 0,
+    BACK_CALCULATION,
+    CONDITIONING_TECHNIQUE,
+    CONDITIONAL_INTEGRATION
+  };
+
+  constexpr AntiwindupStrategy() : value_(NONE) {}
+  explicit constexpr AntiwindupStrategy(Value v) : value_(v) {}
```

Comment:
```suggestion
  constexpr AntiwindupStrategy(Value v) : value_(v) {}  // NOLINT(runtime/explicit)
```

And then the constructor can be omitted and this just works
```
  Pid pid(
    0.0, 1.0, 0.0, 0.0, 0.0, 5.0, -5.0, 1.0, true, false,
    AntiwindupStrategy::BACK_CALCULATION);
```
What do you think? This can be simplified in the tests then

Reply:
That helped a lot. I was unsure whether I could use NOLINT—thanks! Commit e89ac18 addresses this.